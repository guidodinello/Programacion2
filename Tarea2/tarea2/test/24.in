# Implementar subCadena
insertarAlFinal (6,3.5)
subCadena 6 6
insertarAlFinal (6,2.5)
subCadena 6 6
insertarAlFinal (7,2.5)
subCadena 6 6
imprimirCadena
subCadena 7 7
subCadena 6 7
finalCadena
insertarSegmentoDespues 3 (7,3.4) (8,9.4) (9,2.6)
subCadena 6 7
subCadena 7 9
subCadena 6 9
imprimirCadena
Fin
