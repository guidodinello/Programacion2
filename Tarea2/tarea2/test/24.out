1>#  Implementar subCadena.
2>Insertado al final.
3>(6,3.50)
4>Insertado al final.
5>(6,3.50)(6,2.50)
6>Insertado al final.
7>(6,3.50)(6,2.50)
8>(6,3.50)(6,2.50)(7,2.50)
9>(7,2.50)
10>(6,3.50)(6,2.50)(7,2.50)
11>loc al final.
12>Segmento insertado despuÃ©s de loc.
13>(6,3.50)(6,2.50)(7,2.50)(7,3.40)
14>(7,2.50)(7,3.40)(8,9.40)(9,2.60)
15>(6,3.50)(6,2.50)(7,2.50)(7,3.40)(8,9.40)(9,2.60)
16>(6,3.50)(6,2.50)(7,2.50)(7,3.40)(8,9.40)(9,2.60)
17>Fin.
